### DISCUSSION TITLE
Streamline calculation from whole brain tractogram

### DISCUSSION THREAD

**Discussion by gaonkimusc** [dipy/dipy#3195] [Tractography / Tractometry] [answered]
Hello team!

I am wondering if I can get the total number of streamlines in my whole_brain_tractogram.trk file. Is there a command that outputs the total number of streamlines in my trk file? Thank you in advance :)

**Comment by arokem** [comment]
```
from dipy.io.streamline import load_tractogram
tgm = load_tractogram("whole_brain_tractogram.trk")
len(tgm)
```

**Comment by gaonkimusc** [comment]
Hello!
I tried the load_tractogram command but noticed that it leads to a
TypeError: TypeError: load_tractogram() missing 1 required positional
argument: 'reference'

**Reply by arokem** [reply, marked as answer]
Sorry! I got it wrong, try:

```
from dipy.io.streamline import load_tractogram
tgm = load_tractogram("whole_brain_tractogram.trk", reference="same")
len(tgm)
```

**Reply by gaonkimusc** [reply]
This worked! Thank you so much!!

**Reply by skoudoro** [reply]
The cli give you this info also and more !

`dipy_info whole_brain_tractogram.trk`

**Comment by gaonkimusc** [comment]
Thank you!
